src/omero_cli_zarr.py

```diff
@@ -41,32 +45,59 @@ class ZarrControl(BaseControl):
   def _configure(self, parser):
     parser.add_login_arguments()
 
-    parser.add_argument("image_id", type=int, help="The Image to export")
-    parser.add_argument("target", type=str, help="The target directory")
-
-    parser.add_argument("--tile_width", default=None)
-    parser.add_argument("--tile_height", default=None)
-    parser.add_argument("--resolutions", default=None)
-    parser.add_argument("--max_workers", default=None)
+    ProxyStringType("Image")
+    parser.add_argument("object", type=ProxyStringType("Image"),
+      help="The Image to export.")
+    parser.add_argument("--output", type=str, default="", help="The output directory")
+
+    parser.add_argument(
+      "--cache_numpy", action="store_true",
+      help="Save planes as .npy files in case of connection loss")
+
+    parser.add_argument("--bf", action="store_true",
```

Comment:
I could see naming this `--local` and pass it the `ManagedRepository` location directly.